Library/Formula/libsamplerate.rb

```diff
@@ -26,6 +26,19 @@ def install
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
     system "make", "install"
+    rm_f "#{bin}/sndfile-resample" if build.without? "libsndfile"
+  end
+
+  def caveats
+    s = ""
```

Comment:
Let's just stick with that then. If we really need to revisit it, we can do so under a separate PR specific to that change.